packages/indexer/src/EventHandlers.ts

```diff
@@ -278,11 +273,7 @@ ShowHubContract_Funded_handlerAsync(async ({ event, context }) => {
     return
   }
 
-  let conditionModule = await context.ConditionModule.get(entity.conditionModule)
-  if (conditionModule == null) {
-    context.log.error(`ConditionModule ${entity.conditionModule} not found`)
-    return
-  }
+  let conditionModule = await context.Record.getConditionModule(entity)
```

Comment:
Because it is a 'required' (has a `!`) on it in the schema, you can run this function which is how you load linked entities.

Reply:
https://docs.envio.dev/docs/linked-entity-loaders#event-handler-file